server/src/app.ts

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
```

Comment:
We should change to `import * as express from "express"` style imports. I will add TSLint rules for this =]